lib/hsa/hsaRfiMaskOutput.hpp

```diff
@@ -36,8 +36,6 @@ class hsaRfiMaskOutput : public hsaSubframeCommand {
     virtual ~hsaRfiMaskOutput();
     /// Wait for output buffer to be empty, keep track of _rfi_mask_output_buf_precondition_id
     int wait_on_precondition(int gpu_frame_id) override;
-    /// Function to handle updatble config rest server calls for rfi zeroing toggle
-    bool update_rfi_add_lostsamples_flag(nlohmann::json& json);
```

Comment:
was this ever implemented?

Reply:
At one point yes, but looks like when we changed it, this got left behind.